portal/src/locale-data/en.json

```diff
@@ -192,6 +192,15 @@
   "AuthenticationAuthenticator.authenticatorHeader": "Authenticator",
   "AuthenticationAuthenticator.widgetHeader.primary": "Primary Authenticator",
   "AuthenticationAuthenticator.widgetHeader.secondary": "Secondary Authenticator",
+  "AuthenticationAuthenticator.policy.title": "Policy",
+  "AuthenticationAuthenticator.policy.require-mfa": "Require Multi-factor Auth",
+  "AuthenticationAuthenticator.policy.require-mfa.required": "Always Required",
+  "AuthenticationAuthenticator.policy.require-mfa.if-exists": "Required if user has Multi-factor Authenticator",
+  "AuthenticationAuthenticator.policy.require-mfa.if-requested": "Required if user enabled Multi-factor Authenticator",
```

Comment:
I guess we do not support `if_requested` at the moment. Maybe we should hide this option first? @kiootic

Reply:
Can we hide the option in the selection dropdown, but still display a proper localized description in case it is used in configuration? (e.g. updated manually)

Reply:
Yes, found a field called `hidden` in option list

Reply:
Option hidden but showing localized text if manually set in config yaml file

<img width="510" alt="Screenshot 2020-10-08 at 7 01 53 PM" src="https://user-images.githubusercontent.com/54486231/95450371-c897c280-0998-11eb-9bbc-6e87ceac5547.png">